Core/AppRuntime/Include/Babylon/AppRuntime.h

```diff
@@ -14,7 +14,7 @@ namespace Babylon
     {
     public:
         AppRuntime();
-        AppRuntime(std::function<void(std::exception_ptr)> unhandledExceptionHandler);
+        AppRuntime(std::function<void(const Napi::Error&)> unhandledExceptionHandler);
```

Comment:
Why only Napi::Error now? Couldn't you Dispatch a function that just throws an std::exception (for example)?